packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -23,124 +24,50 @@ Front end engineers use end-to-end (e2e) tests in `vets-website` to validate mul
 
 ## End-to-end tests conventions
```

Comment:
I noticed that you ripped out the Nightwatch example spec and didn't replace it with a Cypress example. Should we insert some example Cypress syntax here that demonstrates some of the best practice concepts that you have listed here?

Reply:
Good question, @pjhill . I almost did this but thought it was a bit too in violation of DRY principles and put a link to the example docs rather than repeating them here (especially for a consolidation ticket). 
Being said, if we want maybe a simple example for easy reference I can see the logic in that.

Reply:
Fair point regarding removing redundancy -- let's collect examples in a single spot and link to them from here.